devices/Dac63004/VoutGain.cs

```diff
@@ -0,0 +1,45 @@
+﻿////
+// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+////
+
+namespace Iot.Device.Dac63004
+{
+    /// <summary>
+    /// Selection of Vout gain.
+    /// </summary>
+    public enum VoutGain : byte
+    {
+        // the enum values correspond to the bit values in the register, MSB
+
+        /// <summary>
+        /// Using external reference on VREF pin. Multiply factor is 1x.
+        /// </summary>
+        ExternalRef_1x = 0x0,
```

Comment:
minimal to be consistent with the rest of the enums: you can remove here the first underscore everywhere. The second one is needed to readability for the 1_5x